base/deprecated.jl

```diff
@@ -2093,6 +2093,11 @@ end
     @deprecate chol!(x::Number, uplo) chol(x) false
 end
 
+@deprecate_moved Nullable "Nullables"
```

Comment:
I wonder whether it wouldn't make more sense to deprecate `Nullable` in favor of `Union{Some{T}, Void}` instead. `isnull` would have to be deprecated in favor of `x === nothing`. The problem is that it's not a perfect replacement for `Nullable` so lots of things would break. But it would work for places where people call `isnull` on a value returned e.g. by `tryparse`, when they will currently get a rather misleading message about using `Nullables`.